backend/src/main/kotlin/pl/edu/agh/dockerbuddy/controller/HostController.kt

```diff
@@ -8,6 +8,7 @@ import pl.edu.agh.dockerbuddy.controller.response.ResponseType
 import pl.edu.agh.dockerbuddy.model.entity.Host
 import pl.edu.agh.dockerbuddy.service.HostService
 
+@CrossOrigin
```

Comment:
Do we need this annotation on all endpoints?

Reply:
yes